packages/fast-components-react-msft/src/button/button.schema.json

```diff
@@ -17,7 +17,7 @@
             "title": "Appearance",
             "type": "string",
             "enum": [
```

Comment:
This is spelled correctly in the component though, right?

Reply:
Correct - the reason it showed up correctly the first time was because it was referencing the Enum and the schema was an independent string. Did a search as well and checked similar components (CTA)